tests/test_visitors/test_tokenize/test_consistency/test_string_line_breaks.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    LineCompriseCarriageReturnViolation,
+)
+from wemake_python_styleguide.visitors.tokenize.syntax import (
+    WrongKeywordTokenVisitor,
+)
+
+# Correct:
```

Comment:
Examples are incorrect. Try this:

```python
correct_newline = 'print(1)\nprint(2)'
wrong_newline = 'print(1)\r\nprint(2)'
```

You should also test things like `correct_string = 'my_string = "\r"'`, etc

Reply:
Ah, perhaps today is so late hour :) I'll fix

Reply:
fixed, please check test in this option